.github/semantic.yml

```diff
@@ -0,0 +1,2 @@
+# Documentation: https://github.com/zeke/semantic-pull-requests
```

Comment:
IIRC https://github.com/zeke/semantic-pull-requests GitHub App has been shut down months ago.

You may use https://github.com/Ezard/semantic-prs instead, which keeps the same feature set while having a free service.

Reply:
When I check new PRs in this repo, I don't see the semantic PR check result.

Reply:
@gaocegege @VoVAllen @kemingy Would you like to install this alternative app? https://github.com/apps/semantic-prs